src/components/VHeader/VFilterButton.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <VButton
+    :variant="buttonVariant"
+    class="self-center"
+    :pressed="pressed"
+    aria-controls="filter-sidebar"
+    @click="toggleFilters"
+  >
+    <VIcon
+      v-show="showIcon"
+      :icon-path="filterIcon"
+      :class="{ 'me-2': iconNeedsPadding }"
+    />
+    <span class="filter-label" :class="buttonLabel.a11yProps">{{
+      buttonLabel.text
+    }}</span>
+  </VButton>
+</template>
+
+<script>
+import { computed, defineComponent, useContext } from '@nuxtjs/composition-api'
+import { isScreen } from '~/composables/use-media-query'
+import { useFilterSidebarVisibility } from '~/composables/use-filter-sidebar-visibility'
+
+import filterIcon from '~/assets/icons/filter.svg'
+import VButton from '~/components/VButton.vue'
+import VIcon from '~/components/VIcon/VIcon'
+
+const VFilterButton = defineComponent({
+  name: 'VFilterButton',
+  components: {
+    VIcon,
+    VButton,
+  },
+  props: {
+    /**
+     * Whether the page has been scrolled. If true on mobile, the button
+     * label is blank.
+     * This is currently not implemented, and is always false.
+     * TODO(@obulat): add use-scroll composable to VHeader
+     * @default false
+     */
+    isScrolled: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  setup(props) {
+    const { i18n, store } = useContext()
+    const { isScrolled = false } = props
```

Comment:
Destructuring the `props` object will remove reactivity from the properties and their values will get stuck on the first value. Learned that one the hard way! 😭